app/services/metadata_service.rb

```diff
@@ -39,8 +39,22 @@ def local_attribution_list
       values
     end
 
-    def format_list
-      ['Cartographic', 'Data', 'Image', 'Notated music', 'Text', 'Video', 'Sounding recordings-nonmusical']
+    def resource_type_list
```

Comment:
@mcritchlow : Yes, I agree. I think one way that we can do is to add some special class like "ResourceType" to extend skos:Concept or the basic Authority class (with label and URI only), then we can simply management them through the local authority management system.